README.md

```diff
@@ -35,219 +31,101 @@ The SDK simplifies the setup and interaction with SAP AI Core, allowing you to f
-By default, the SDK automatically extracts these credentials from a service instance of  type `aicore` bound to your application.
-
-If running the application locally without this service binding, you may encounter an exception:
-
-```
-Could not find any matching service bindings for service identifier 'aicore'
-```
-
-There are multiple ways to provide these credentials:
-
-| Option | Description                                                                                              |
-|--------|----------------------------------------------------------------------------------------------------------|
-| **1**  | Create an `.env` file containing an `AICORE_SERVICE_KEY={...}`                                           |
-| **2**  | Regular service binding in SAP BTP Cloud Foundry (results in `VCAP_SERVICES` environment variable entry) |
-| **3**  | Define and use a _Destination_ in the SAP BTP Destination Service                                        |
-
-Additional methods (not recommended for production):
-
-- Use the [hybrid testing](https://cap.cloud.sap/docs/advanced/hybrid-testing#services-on-cloud-foundry) approach for
-  CAP applications (e.g., `cds bind --to aicore --exec mvn spring-boot:run`)
-- Leverage a "user-provided" service binding
-- Define and use a custom `ServiceBinding` or `ServiceBindingAccessor` in your application
-
-### Option 1: Set AI Core Credentials
-
-<details>
-<summary>Click to view detailed steps</summary>
-
-
-**1. Obtain Service Credentials:**
-
-- Log into the **SAP BTP Cockpit**
-- Navigate to **Services** -> **Instances and Subscriptions** -> **Instances** -> **AI Core**
-- Click **View Credentials** and copy the JSON content
-
-**2. Create `.env` file:**
-
-- Create an `.env` file in the root directory of your application
-- Add a **one line** entry `AICORE_SERVICE_KEY={...}` with the copied JSON
-
-<details>
-<summary>Set an environment variable instead of .env</summary>
+## Getting Started
 
-**2. Set an Environment Variable: (alternative)**
+### What You'll Build
 
-- In your IDE or terminal, set the environment variable `AICORE_SERVICE_KEY` with the copied JSON content
+In this quickstart, you'll use the OpenAI GPT-4o model through the [Orchestration Service of AI Core](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration) for generating text.
+The application will send a prompt to the AI model and display the generated response.
 
-  ℹ️ The environment variable has priority over the `.env` file.
+### Prerequisites
 
-Example Linux/MacOS:
+This quickstart assumes you have a **deployment of the Orchestration service available** in the `default` resource group of your AI Core instance.
+If you don't have a deployment yet, please refer to [this guide](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-orchestration) on how to create one.
```

Comment:
I intentionally didn't add the code to create the config + deployment using our SDK APIs. Doing that in Java code is somewhat tedious and has the danger that it is executed more than once, creating unnecessary / duplicate configurations / deployments.

But more strongly, this step will become obsolete soon (hopefully) with a default orchestration deployment being implemented.